docs/components/networkmanager.md

```diff
@@ -109,6 +110,12 @@ public void Disconnect()
 }
 ```
 
+### Shutting Down
```

Comment:
```suggestion
### Shut down a network
```

Reply:
Shutting Down NetworkManager?

Shut down a network seems like you are shutting down an entire network.

Reply:
Possibly:
Shutting down a network session